### PR TITLE
[WIP] This is the gui part of the commit where the wire feed is refreshed

### PR DESCRIPTION
**Pull request by PYRET1C**
The wire feed is refreshed if there is any event published.

### CODE REVIEW

**Comment by defnax** [comment]
please update your branch, not using latest master :)

**Comment by PYRET1C** [comment]
yes i will do it soon.

**Comment by PYRET1C** [comment]
Hello defnax,
I made some changes and tested them. please test them if they are working.
I updated the branch to master.

**Comment by defnax** [comment]
Now it works accounts updated after created

**Comment by PYRET1C** [comment]
Awesome, thanks! I will edit and remove some comments. I also have to add
some comments. And can you test it a few more times or are you satisfied
with it? I just wanted to see If there is any crash happening after
starting retro share.

**Comment by defnax** [comment]
youcan correct/clean this one 

> //    UPDATED_MESSAGE                 = 0x04, // existing message has been updated in a particular channel
//    RECEIVED_PUBLISH_KEY            = 0x05, // publish key for this channel has been received
    FOLLOW_STATUS_CHANGED        = 0x03, // subscription for channel mChannelGroupId changed.
    POST_UPDATED                 = 0x04, // subscription for channel mChannelGroupId changed.
//    READ_STATUS_CHANGED             = 0x07, // existing message has been read or set to unread
//    RECEIVED_DISTANT_SEARCH_RESULT  = 0x08, // result for the given group id available for the given turtle request id
//    STATISTICS_CHANGED              = 0x09, // stats (nb of supplier friends, how many msgs they have etc) has changed
//    SYNC_PARAMETERS_UPDATED         = 0x0a, // sync and storage times have changed
    NEW_REPLY                     = 0x04, // new comment arrived/published. mChannelThreadId gives the ID of the commented message
    NEW_LIKE                       = 0x05, // new vote arrived/published. mChannelThreadId gives the ID of the votes message comment
    NEW_REPUBLISH                 = 0x06, // channel was deleted by auto-cleaning system
    NEW_REPLY                     = 0x05, // new comment arrived/published. mChannelThreadId gives the ID of the commented message
    NEW_LIKE                       = 0x06, // new vote arrived/published. mChannelThreadId gives the ID of the votes message comment
    NEW_REPUBLISH                 = 0x07, // channel was deleted by auto-cleaning system

**Comment by defnax** [comment]
@csoler

File: retroshare-gui/src/gui/TheWire/WireDialog.cpp [hunk 1]
```diff
@@ -91,6 +90,54 @@ WireDialog::WireDialog(QWidget *parent)
...
+#ifdef GXSWIRE_DEBUG
+                RsDbg() << " Refreshing the feed if there is a matching event. "<< std::endl;
+#endif
+
+        // The following switch statements refresh the wire feed whenever there is a new event
+        switch(e->mWireEventCode)
+        {
+        case RsWireEventCode::NEW_POST:
+            refreshGroups();
+            break;
```
Comment (csoler) [inline retroshare-gui/src/gui/TheWire/WireDialog.cpp]: For code readability and simplification, plz remove the break and calls to refreshGorup(), except the last one.

Comment (PYRET1C) [reply retroshare-gui/src/gui/TheWire/WireDialog.cpp]: Please take a look at them, i did it.

**Pull request merged**